Games/core/VegKickMeeting.js

```diff
@@ -12,6 +12,9 @@ module.exports = class VegKickMeeting extends Meeting {
         this.inputType = "button";
         this.targets = ["Kick"];
         this.votesInvisible = true;
+        // when enough kicks has reached, people who have not voted can spam kick
```

Comment:
This is because the game does not accept the person's vote as valid, allowing them to vote an unlimited number of times.